README.md

```diff
@@ -58,84 +58,131 @@ Here is an overview of all fields that can be present in plugin meta YAML files
 
 |field name in YAML | description | mandatory |
 |-------|-------|-------|
-| id | plugin ID | yes |
+| publisher | publisher name | yes |
```

Comment:
I think it would be better to put `publisher`, `name`, `version` fields description together

Reply:
done